### PR TITLE
Update CVE-2022-24874.json with rejection

### PR DESCRIPTION
**Pull request by shelbyc**
Reject CVE-2022-24874 as a duplicate of CVE-2022-28820.

### CODE REVIEW

File: 2022/24xxx/CVE-2022-24874.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
```
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24874.json]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24874.json]: ('Invalid control character at: line 4 column 32 (char 110)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/24xxx/CVE-2022-24874.json (hunk 1 above)
Comment (shelbyc) [reply 2022/24xxx/CVE-2022-24874.json]: This was the smart quote that was causing issues.

**Pull request merged**